ConfigManager.cs

```diff
@@ -14,8 +15,8 @@ public class Config
 		string path = "./config.json";
 		if (!File.Exists(path))
 		{
-			Program.LogAsync("Config", $"Config file not found at: {path}", LogSeverity.Critical);
-			return null;
+            Log.Fatal("Config file not found at: {Path}", path);
```

Comment:
The message doesn't actually return a path, but just says "./config.json", which doesn't help fix the problem
I think I actually forgot to test it in my original code, so I guess it can be fixed now :P

```suggestion
            Log.Fatal($"Config file not found at: {Path.GetFullPath(path)}");
```